pandas/tests/test_downstream.py

```diff
@@ -131,3 +132,31 @@ def test_pyarrow(df):
     table = pyarrow.Table.from_pandas(df)
     result = table.to_pandas()
     tm.assert_frame_equal(result, df)
+
+
+def test_missing_required_dependency(monkeypatch):
+    original_import = __import__
+
+    def mock_import_fail(name, *args, **kwargs):
+        if name == "numpy":
+            raise ImportError("cannot import name numpy")
+        elif name == "pytz":
+            raise ImportError("cannot import name some_dependency")
+        elif name == "dateutil":
+            raise ImportError("cannot import name some_other_dependency")
+        else:
+            return original_import(name, *args, **kwargs)
+
+    expected_msg = (
+        "Unable to import required dependencies:"
+        "\nnumpy: cannot import name numpy"
```

Comment:
quickly looking at one of the ci failures, i think that you need to make these raw strings since the match parameter of pytest.raises assumes a regex.

Reply:
It was even simpler than that - following a change to formatting the error message, it wasn't putting the newline before the first numpy error.